src/components/MultiBarToggle.js

```diff
@@ -115,6 +115,9 @@ class MultiBarToggle extends Component {
                 <Animated.View
                     key={`action_${i}`}
                     style={[Styles.actionContainer, {
+                        flexDirection: 'column',
```

Comment:
Move style properties to the styles.

Reply:
actionContainer: {
        position: 'absolute',
        flexDirection: 'column',
        justifyContent: 'center',
        alignItems: 'center',
    },